features/go.mod

```diff
@@ -4,20 +4,22 @@ go 1.20
 
 require (
 	github.com/google/uuid v1.3.1
-	github.com/temporalio/features/harness/go v1.0.0
+	github.com/temporalio/features/harness/go v0.0.0-00010101000000-000000000000
 	github.com/uber-go/tally/v4 v4.1.1
-	go.temporal.io/api v1.26.1
-	go.temporal.io/sdk v1.25.2-0.20231129171107-288a04f72145
-	go.temporal.io/sdk/contrib/tally v0.2.1-0.20231129171107-288a04f72145
+	go.temporal.io/api v1.26.1-0.20240103185939-608bdd111e4b
+	go.temporal.io/sdk v1.25.1
```

Comment:
Looks like you went back to 1.25.1 vs keeping at `@master` for SDK which may be breaking the build (didn't check)